### PR TITLE
add udwSqlite3

### PR DESCRIPTION
**Pull request by rogercarter1**

### CODE REVIEW

File: udwSqlite3/0sql.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+)
+
+func (db *Db) MustQuery(req QueryReq) {
+	errMsg := db.Query(req)
+	if errMsg != "" {
+		panic(errMsg)
+	}
+}
+
+func (db *Db) GetAllDataInTableToRowMap(tableName string) []map[string]string {
```
Comment (smallsmallwolf) [inline udwSqlite3/0sql.go]: function name should have prefix 'Must'

File: udwSqlite3/0sql.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	return db.mustQueryToMapRowList("SELECT * FROM " + mustEscapeTableOrDatabaseName(tableName) + " ")
+}
+
+func (db *Db) GetTableNameList() []string {
```
Comment (smallsmallwolf) [inline udwSqlite3/0sql.go]: function name should have prefix 'Must'

File: udwSqlite3/0sqlL1.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+			db.cClose()
+			DeleteSqliteDbFileByPath(db.req.FilePath)
+			db.mustInitDbL1()
+			return true
+		}
+	}
+	return false
+}
+
+func DeleteSqliteDbFileByPath(path string) {
```
Comment (smallsmallwolf) [inline udwSqlite3/0sqlL1.go]: function name should have prefix 'Must'

File: udwSqlite3/0sql.go (hunk 2 above)
Comment (daniel-henderson-ty) [inline udwSqlite3/0sql.go]: function name should have prefix 'Must'

File: udwSqlite3/0sql.go (hunk 1 above)
Comment (daniel-henderson-ty) [inline udwSqlite3/0sql.go]: function name should have prefix 'Must'

File: udwSqlite3/0sql.go [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+			output = append(output, string(row[0]))
+		},
+		ColumnCount: 1,
+	})
+	if errMsg != "" {
+		panic(errMsg)
+	}
+	sort.Strings(output)
+	return output
+}
```
Comment (daniel-henderson-ty) [inline udwSqlite3/0sql.go]: function name should have prefix 'Must'

**Pull request merged**